docs/proposals/PROPOSAL_C.md

```diff
@@ -0,0 +1,175 @@
+  - Defines namespace for their community metadata defined in artifacts
+  - Develops compatibility checking tools
+- A **Compatibility Specification** is a pairing of metadata and method to assess goodness of fit
+  - Maintained by a compatibility interest group
+  - Method needs to return a boolean outcome "yes" or "no"
+  - Metadata needs to be namespaced to a prefix owned by a CIG.
+- A **Namespace** is a named identifies that provides an organization under which users can specify compatibility objects.
+  - Owned and registered to a specific CIG
+- A **Compatibility Plugin** conforms to an interface, and is developed by compatibility interest groups to run easily on the host, either integrated into a runtime (or similar) tool, or as a standalone tool.
+
+The new image compatibility specification includes a JSON compatibility structure with keys that are owned by compatibility interest groups, and include simple key, value pairs that can be harnassed by downstream tools for analysis of compatibility. Importantly, there is no complex graph structure in the JSON object itself, but rather, the tools can supply the graph. This gives power and freedom to the compatibility interest group developers to create tools as they see fit. Several designs are afforded:
+
+1. **Simple checking of key/value pairs**: A community might have an algorithm that is as simple as their tool checking for particular keys, and that values match for a host of interest.
+2. **Graph-based logic**: A community might warrant using advanced graph parsing to do a similar assessment. An example comes from [Archspec](https://github.com/archspec), where a single architecture (from a flat artifact or the platform, depending on what the tool chooses) can [be placed in a graph](https://github.com/archspec/archspec-json/blob/master/cpu/microarchitectures.json) and used for advanced assessment of compatibility.
+
+The above design encourages simplicity and designation of fewer metadata pieces in the OCI objects. Any complex graph-based query or additional relationships should be represented on the level of the tools that do the compatibility checking. This ensures that changes to these methods or graphs do not require re-creating extensive numbers of artifacts.
+
+## Modifications
+
+This proposal is agnostic to where the JSON object is placed. There are arguably pros and cons to different approaches, and the author (vsoch) leaves it to the expertise of the compatibility working group to decide on a "best" approach.
+
+### Image Manifest
+
+The JSON object could go directly in the image manifest or list to not require any additional artifact query or queries. For runtime tools, this would mean (still) just one call to retrieve all needed metadata to do a check. There would be no additional calls to retrieve plugins. Any compatibility checking plugins (discussed later in this proposal) would already be vetted and installed alongside the runtime tools.
```

Comment:
Do you consider container runtimes (high-level, as containerd, docker etc.) as part of the runtime tools category?

Reply:
I recall that somebody on the meeting said that container runtime implementations may reject too big manifests. I am afraid if we put in the index or image manifest the compatibility requirement it may be rejected. This can be discussed on the meeting.